js/container/InputContainer.js

```diff
@@ -0,0 +1,23 @@
+import {createTodoItem} from "../api/index.js";
+import {addTodoItem} from "../reducer.js";
+
+function InputContainer($dom, store) {
+    $dom.addEventListener('keyup', async ({target, key})=>{
+        switch (key){
+            case 'Enter':{
+                try {
+                    const {selectedUserId} = store.getState();
+                    const todoItem = await createTodoItem(selectedUserId, target.value);
+                    store.dispatch(addTodoItem({todoItem}));
+                    target.value = '';
+                }catch (e) {
+                    alert(e);
+                    console.error(e)
+                    target.value = '';
+                }
```

Comment:
```suggestion
                try {
                    const {selectedUserId} = store.getState();
                    const todoItem = await createTodoItem(selectedUserId, target.value);
                    store.dispatch(addTodoItem({todoItem}));
                }catch (e) {
                    alert(e);
                    console.error(e)
                }
                 target.value = '';
```
공통된 부분은 밖으로 빼면 좋을 것 같아요.
case 가 Enter 하나 뿐인데 switch 를 쓴 특별한 이유가 있나요?